wdl-parsing/WDLParsingFunction/checkstyle.xml

```diff
@@ -0,0 +1,424 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+        "https://checkstyle.org/dtds/configuration_1_3.dtd">
+
+<!--
```

Comment:
I'll add a comment in both checkstyle files, but I think this should be the gold copy since it's more strict and fully compatible with the Google codestyle (instead of the dockstore/dockstore-modified one)

Reply:
That works too